### PR TITLE
[solid3] fix arm64-osx build

### PR DESCRIPTION
**Pull request by autoantwort**

### CODE REVIEW

File: ports/solid3/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,17 @@
 {
   "name": "solid3",
-  "version-string": "3.5.8",
+  "version": "3.5.8",
   "port-version": 1,
```
Comment (JackBoosY) [inline ports/solid3/vcpkg.json:4]:
```suggestion
  "port-version": 2,
```

File: ports/solid3/no-sse.patch [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -87,8 +87,8 @@ if(MSVC)
+ endif(MSVC)
+ 
+ if(UNIX)
+-  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -ffast-math -msse2 -mfpmath=sse")
+-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-rtti -Wall -ffast-math -msse2 -mfpmath=sse")
++  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -ffast-math")
++  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-rtti -Wall -ffast-math")
```
Comment (JackBoosY) [inline ports/solid3/no-sse.patch:12]: I think it's incorrect? Any evidence that arm64 darwin doesn't support sse2?

**Review by JackBoosY — CHANGES_REQUESTED** [review]

File: ports/solid3/no-sse.patch (hunk 2 above)
Comment (autoantwort) [reply ports/solid3/no-sse.patch:12]: SSE in an x86 extension. arm has neon.

Comment (JackBoosY) [reply ports/solid3/no-sse.patch:12]: So x64-linux / x86-linux should keep this, right?
Can you please report this to the upstream? I would like to know upstream opinion.

Comment (autoantwort) [reply ports/solid3/no-sse.patch:12]: It is already enabled by default: https://stackoverflow.com/a/30718552/10162645

Comment (JackBoosY) [reply ports/solid3/no-sse.patch:12]: We'd better make it clear, what about x86?

Comment (JackBoosY) [reply ports/solid3/no-sse.patch:12]: Ping for response.

Comment (autoantwort) [reply ports/solid3/no-sse.patch:12]: Done

File: versions/s-/solid3.json [hunk 3]
```diff
@@ -1,8 +1,13 @@
...
     {
-      "git-tree": "1f494d1d6f610d9c26444a12f2b9952f7485601d",
-      "version-string": "3.5.8",
+      "git-tree": "d02f4f19962318fca6af9450781fcd281af86652",
+      "version": "3.5.8",
+      "port-version": 2
+    },
+    {
+      "git-tree": "13971c5e65420abecb393378cd59d7d9eab111b3",
+      "version": "3.5.8",
```
Comment (JackBoosY) [inline versions/s-/solid3.json:10]:
```suggestion
      "git-tree": "1f494d1d6f610d9c26444a12f2b9952f7485601d",
      "version-string": "3.5.8",
```

**Review by JackBoosY — APPROVED** [review]

**Pull request merged**